pages/index.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="container">
+    <div>
+      <logo />
+      <h1 class="title">
+        DrinkingCast
+      </h1>
+      <h2 class="subtitle">
+        A drinking game for chromecast.(temporary name, ready still WIP)
+      </h2>
+      <div class="links">
+        <a
+          href="https://nuxtjs.org/"
+          target="_blank"
+          class="button--green"
+        >
+          Documentation
+        </a>
+        <a
+          href="https://github.com/nuxt/nuxt.js"
+          target="_blank"
+          class="button--grey"
+        >
+          GitHub
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Logo from '~/components/Logo.vue'
+
+export default {
+  components: {
+    Logo
+  }
+}
+</script>
+
+<style>
```

Comment:
out of interest:
moet de style in dezelfde file staan? of zouden we dit later ook kunnen splitsen naar andere files?

Reply:
Dat wilde ik je gaan uitleggen als ik je join maar voor nu ff. Zoals je miss al wist, Vue.JS werkt met components. Elk component heeft zn eigen template, zijn eigen style en zijn eigen script (js). De css van een component staat 'scoped' dat houdt in dat hij alleen voor dat component geldt. Een 'page' (dat is een element van het framework Nuxt. Leg ik je nog uit) implenteert deze Components en heeft daarnaast nog een style voor de webapp waar je de components 'in plaatst'. Deze is niet scoped omdat deze style voor de hele webapp geldt. Wat we nog gaan doen is Sass installeren/toevoegen. Sass is een css-language waarmee je variables (fonts, colors) kan definen om ze vervolgens in de app te verwerken.

Reply:
ahh oke, helder. Thanks voor de uitleg.